### ISSUE TITLE
Torrentbytes not working

### ISSUE THREAD

**Issue by gres147679** [zone117x/Jackett#189]
When trying to add torrentbytes as an indexer the UI gives the following error:

` Configuration failed: Found no results while trying to browse this tracker`

The backend fails with the following stacktrace:

```
09-27 18:36:33 Info Found 0 releases from TorrentBytes
09-27 18:36:33 Error Exception in Configure System.Exception: Found no results while trying to browse this tracker
  at Jackett.Services.IndexerManagerService+<TestIndexer>d__9.MoveNext () [0x00000] in <filename unknown>:0
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x00000] in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () [0x00000] in <filename unknown>:0
  at Jackett.Controllers.AdminController+<Configure>d__15.MoveNext () [0x00000] in <filename unknown>:0
```

**Comment by poplolnman** [comment]
im having the same issues with bitmetv

**Comment by poplolnman** [comment]
also for animetorrents.me

**Comment by Kayomani** [comment]
Fixed in released v0.6.7
